hnn_core/tests/test_cell.py

```diff
@@ -13,14 +13,6 @@
 
 def test_cell():
     """Test cells object."""
-    hnn_core_root = op.dirname(hnn_core.__file__)
-    params_fname = op.join(hnn_core_root, 'param', 'default.json')
-    params = read_params(params_fname)
-
-    net = Network(params)
-    with NetworkBuilder(net) as neuron_net:
-        neuron_net.cells[0].plot_voltage()
```

Comment:
@ntolley I think this is the change that caused Travis to fail. Since `load_custom_mechanisms()` is currently only called within a `NetworkBuilder` instance, it needs to be called separately in order to successfully instantiate an `_ArtificialCell` object. If you import `load_custom_mechanisms` from `hnn_core.network_builder` and run it at the beginning of test_cell.py, pytest runs perfectly.

Reply:
Really appreciate the help with this, managed to reproduce locally and this immediately resolved the error.

Should we consider an alternative place the `load_custom_mechanisms()` call? I'm not sure how frequently this sort issue would turn up outside of writing tests, but it seems like a non-intuitive bug to trace.

Reply:
maybe in the init.py file, not sure ... I'd suggest trying this in a separate PR though. Make the tests happy here so we can move on